react/components/SearchBar/components/SearchBar.js

```diff
@@ -1,20 +1,21 @@
-import React, { useRef, useCallback } from 'react'
+import React, { useRef, useCallback, useState, useEffect } from 'react'
 import PropTypes from 'prop-types'
 import classNames from 'classnames'
 import Downshift from 'downshift'
 import { NoSSR } from 'vtex.render-runtime'
 import { Overlay } from 'vtex.react-portal'
 import { useRuntime } from 'vtex.render-runtime'
 import { useCssHandles } from 'vtex.css-handles'
+import debounce from 'debounce'
```

Comment:
⛏(nitpick) I would move this `import` to line 5, just to keep all imports from packages that are not inside of IO together.